tsconfig.json

```diff
@@ -26,13 +26,29 @@
     "resolveJsonModule": true,
     "emitDeclarationOnly": true,
     "baseUrl": ".",
-    "composite": true
+    "composite": true,
+    "paths": {
+      "multiformats": [
+        "./src/index.js"
+      ],
+      "multiformats/interface": [
+        "./src/interface"
+      ],
+      "multiformats/hashes/interface": [
+        "./src/hashes/interface"
+      ],
+      "multiformats/*": [
+        "./src/*.js"
+      ]
+    }
   },
   "include": [
-    "src"
+    "src",
+    "test"
```

Comment:
Was this change necessary? I think it might be what was busting the ts-use code.